### ISSUE TITLE
[Question]: The results of multiple calculations are exactly the same

### ISSUE THREAD

**Issue by chanchanchanchanchan** [Valdes-Tresanco-MS/gmx_MMPBSA#506]
### My Question is...

Dear Developers,
I have changed my force field to calculate the free energy of my polypeptide and membrane system. The first time, the free energy was calculated using the AMBER force field, while the second time, the free energy was calculated using the force field employed in the simulation. The calculation results are in the attached image.
Thank you for your time!
![result_1](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/assets/103733575/a9574f05-8d39-4bb0-b9cf-d4942f387519)
![result_2](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/assets/103733575/fd60f408-f36b-4da3-b69f-923134c17787)

**Comment by marioernestovaldes** [comment]
which command lien did you use?

**Comment by chanchanchanchanchan** [comment]
> which command lien did you use?

The command line I use is ”mpirun -np 10 gmx_MMPBSA -O -i ../mmpbsa.in -cs ../pcpyy-mycelium-md320ns-pcpyy-mem.tpr -ci ../index-md320ns.ndx -cg 21 20 -ct ../pcpyy-mycelium-md320ns-pcpyy-mem.xtc -cp ../topol-md320ns-MMPBSA.top -o pcpyy-mycelium-md320ns_MMPBSA.dat -eo pcpyy-mycelium-md320ns_MMPBSA.csv -nogui“

**Comment by marioernestovaldes** [comment]
When you define the topology (as recommended) in the command line, the force fields variable is ignored and the amber topology is generated using the one that was defined in the command line, hence giving the same results as expected.

**Comment by chanchanchanchanchan** [comment]
> When you define the topology (as recommended) in the command line, the force fields variable is ignored and the amber topology is generated using the one that was defined in the command line, hence giving the same results as expected.

How do I use the force field employed in my simulation?

**Comment by chanchanchanchanchan** [comment]
> When you define the topology (as recommended) in the command line, the force fields variable is ignored and the amber topology is generated using the one that was defined in the command line, hence giving the same results as expected.

By the way, the force field I employed in mu simulaition is Charmm force field.

**Comment by marioernestovaldes** [comment]
The topology you are defining in the command line should contain all the parameters according to the charm force field if you used charm force field to build the topology in the first place... so, as long as that's the topology you used to run the simulation, there shouldn't be any issues

**Comment by chanchanchanchanchan** [comment]
> The topology you are defining in the command line should contain all the parameters according to the charm force field if you used charm force field to build the topology in the first place... so, as long as that's the topology you used to run the simulation, there shouldn't be any issues

I have defiend "-cs ../pcpyy-mycelium-md320ns-pcpyy-mem.tpr" in the command line (and this is the topology I used to run the simulation). Does this mean I am using my original force field?
The first time I ran it, I specified forcefields="oldff/leaprc.gaff" in the mmpbsa.in file and used this command "-cs ../pcpyy-mycelium-md320ns-pcpyy-mem.pdb". The second time I delete forcefields="oldff/leaprc.gaff" in the mmpbsa.in file and used this command "-cs ../pcpyy-mycelium-md320ns-pcpyy-mem.tpr".
